lib/controllers/remote-controller.js

```diff
@@ -69,10 +70,13 @@ export default class RemoteController extends React.Component {
     createPrUrl += '/compare/' + encodeURIComponent(currentBranch.getName());
     createPrUrl += '?expand=1';
 
-    return new Promise((resolve, reject) => {
-      shell.openExternal(createPrUrl, {}, err => {
-        if (err) { reject(err); } else { resolve(); }
-      });
-    });
+    let returnPromise;
```

Comment:
@smashwilson, does this look right to you?

I was having trouble getting the inner promise to return for the test. Other than adding the counter, this should behave identically.

Reply:
that was the first thing I tried, and the promise never returned in the test.

I also tried using a .done() callback in the test and the promise still did not return.

Lemmesee if I can figure out a different approach.